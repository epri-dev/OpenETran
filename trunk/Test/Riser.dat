* here is a first line comment

1 2 30.0 0 0 0.01e-6 1.0e-6

cable 1 30.0 1.5e8 0.0e3

* surge input here
* second comment in a row

* third comment in a row

labelpole 1 riser
labelpole 2 opntie

surge -10.0e3 1.0e-6 50.0e-6 0.0e-6
pairs 1 0
* poles 2
poles 1

arrester 5.0e3 5.0e3 0.5 1.0e-6 0.0
pairs 1 0
poles 1

resistor 400.0
pairs 1 0
poles 1

meter
pairs 1 0
poles all


* comment on the last line
