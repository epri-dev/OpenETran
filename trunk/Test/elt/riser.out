  N   span     dT   Tmax
  1  30.00 1e-008 1e-006
Z-phase
  3.00000e+001
Z-modal
  3.00000e+001
Modal Transformation (Ti)
  1.00000e+000
Meter at pole 1, from 1 to 0, max V   = -8.934467e+003
Meter at pole 2, from 1 to 0, max V   = -1.167293e+004
Arrester at pole 1, from 1 to 0 discharged -7.868935e+003 Amperes.
	Time operated: 1.300000e-007	Time of peak: 1.000000e-006
	Charge: -3.206461e-003	Energy: 2.465809e+001
nr_iter = 0, nr_max = 0
Execution time [sec]: 0
